addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -262,6 +262,7 @@ export class OdooEditor extends EventTarget {
                 allowInlineAtRoot: false,
```

Comment:
This commit will be extracted to [this other PR](https://github.com/odoo/odoo/pull/141412).